app/resources/en/translation/shutterpage.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Apply for probate - Eligibility",
+  "header": "Sorry, this service is temporarily unavailable",
+  "introduction": "We&rsquo;re working to improve the service. Please try again later.",
+  "paragraph1": "If you&ve started an application then your information has been saved in your account.",
+  "paragraph2": "You can contact us if you need help with your application.",
+  "paragraph3": "You can also <a rel=\"external noopener\" href=\"http://hmctsformfinder.justice.gov.uk/HMCTS/GetForm.do?court_forms_id=735\">download an application form</a>, print it out and complete it.",
+  "contactHeader": "Contact Us",
+  "paragraph4": "Phone: 03003003648 (Monday to Friday, 9am to 5pm",
```

Comment:
1. There is a typo in the phone number, which should be 0300 303 0648.

2. There should be a closing bracket at the end of the line.

We have the helpline number and opening hours in the config file so it would probably be better to get those from there rather than defining them again in the content file.